src/main/java/bio/terra/workspace/common/utils/ControllerValidationUtils.java

```diff
@@ -79,4 +80,10 @@ public static void validateCloudContext(CloudContext context) {
           "Invalid cloud context. Currently, only Google contexts are supported.");
     }
   }
+
+  public static void validateGoogleBucket(GoogleBucketCreationParameters params) {
+    if (params.getName() == null) {
```

Comment:
I'm not sure how much validation I get for free from the generated code's required attributes.